python/ray/air/checkpoint.py

```diff
@@ -38,6 +40,12 @@
 logger = logging.getLogger(__name__)
 
 
+class CheckpointDict(dict):
```

Comment:
This needs an API annotation - I think developer API should be fine.
```suggestion
@DeveloperAPI
class CheckpointDict(dict):
```